.github/workflows/NotEnoughUpdates-REPO-Workflow.yml

```diff
@@ -2,17 +2,20 @@ name: NotEnoughUpdates REPO Workflow
 
 # Controls when the action will run. 
 on:
-  pull_request_target:
-    branches: [ master ]
   push:
     branches: [ master ]
+  pull_request_target:
+    branches: [ master ]
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      if: ${{ github.event_name == "pull_request_target" }}
```

Comment:
the issue is you now need a second actions/checkout with if event_name == push

Reply:
otherwise it wouldn't checkout on push

Reply:
If I understand this correctly it should always do "uses: actions/checkout@v2" and then only when it's a pr do "with ref"

Reply:
Watching you fight is funny

Reply:
> Watching you fight is funny

😡

Reply:
> If I understand this correctly it should always do "uses: actions/checkout@v2" and then only when it's a pr do "with ref"

you don't
and if block decides whether or not to run the entire thing (so if the if block is false, actions/checkout will not be run, but skipped instead)